lib/zeebe_bpmn_rspec/activated_job.rb

```diff
@@ -21,8 +21,13 @@ def initialize(job, type:, process_instance_key:, client:, context:, validate:)
         context.instance_eval do
           expect(job).not_to be_nil, "expected to receive job of type '#{type}' but received none"
           aggregate_failures do
-            expect(job.processInstanceKey).to eq(process_instance_key)
-            expect(job.type).to eq(type)
+            unless called
```

Comment:
I think putting `unless called` inside `if validate` might be a little confusing, both in terms of code structure and also for users of this API (since `validate` and `called` are bound in a non-obvious way). 

What if we were to separate and rename both `validate` and `called` into new options that spell out exactly what they do and have sensible defaults? The `validate` option could perhaps be `presence: true`, and `called: false` could be `main_instance: true`, or something like that. Then there could be separate `if presence` and `if main_instance` blocks.